python/iot3/src/iot3/core/mqtt.py

```diff
@@ -46,6 +52,48 @@ def __init__(
                             msg_cb function.
         :param span_ctx_cb: The function to obtain a context manager for an
                             OpenTelemetry span.
+
+        There are two ways to connect to the MQTT broker:
+         * via TCP: both host and port must be specified;
+         * via websockets: all of host, port, and websocket_path must
```

Comment:
`all of the host, ` ?

Reply:
Not really, no. It must be read as: _all the items in the following list: host, port, and websocket_path_.